.idea/runConfigurations/JabRef_Main.xml

```diff
@@ -2,6 +2,7 @@
   <configuration default="false" name="JabRef Main" type="Application" factoryName="Application" singleton="false">
     <option name="MAIN_CLASS_NAME" value="org.jabref.JabRefMain" />
     <module name="JabRef.main" />
+    <option name="REDIRECT_INPUT" value="true" />
```

Comment:
This is not related to this PR, is it?

Reply:
Not directly...was added by intellj automatically.